aspnetcore/security/authentication/cookie.md

```diff
@@ -14,335 +14,369 @@ uid: security/authentication/cookie
 
-To sign out the current user and delete their cookie, call the following inside your controller:
+## Signing out
 
 # [ASP.NET Core 2.x](#tab/aspnetcore2x)
 
+To sign out the current user and delete their cookie, call [SignOutAsync](/dotnet/api/microsoft.aspnetcore.authentication.authenticationhttpcontextextensions.signoutasync?view=aspnetcore-2.0):
+
 ```csharp
-await HttpContext.SignOutAsync("MyCookieAuthenticationScheme");
+await HttpContext.SignOutAsync(
+    CookieAuthenticationDefaults.AuthenticationScheme);
 ```
 
 # [ASP.NET Core 1.x](#tab/aspnetcore1x)
 
+To sign out the current user and delete their cookie, call [SignOutAsync](/dotnet/api/microsoft.aspnetcore.authentication.authenticationhandler-1.signoutasync?view=aspnetcore-1.1):
+
 ```csharp
-await HttpContext.Authentication.SignOutAsync("MyCookieAuthenticationScheme");
+await HttpContext.Authentication.SignOutAsync(
+    CookieAuthenticationDefaults.AuthenticationScheme);
 ```
 
 ---
 
+If you aren't using `CookieAuthenticationDefaults.AuthenticationScheme` (or "Cookies") as the scheme (for example, "ContosoCookie"), supply the scheme you used when configuring the authentication provider.
+
 ## Reacting to back-end changes
 
->[!WARNING]
-> Once a principal cookie has been created, it becomes the single source of identity. Even if you disable a user in your back-end systems, the cookie authentication has no knowledge of this, and a user stays logged in as long as their cookie is valid.
+Once a cookie is created, it becomes the single source of identity. Even if you disable a user in your back-end systems, the cookie authentication system has no knowledge of this, and a user stays logged in as long as their cookie is valid.
 
-The cookie authentication provides a series of events in its option class. The `ValidateAsync()` event can be used to intercept and override validation of the cookie identity.
+The `ValidateAsync()` event can be used to intercept and override validation of the cookie identity. This approach mitigates the risk of unauthorized access to an app.
 
-Consider a back-end user database that may have a "LastChanged" column. In order to invalidate a cookie when the database changes, you should first, when [creating the cookie](xref:security/authentication/cookie#security-authentication-cookie-middleware-creating-a-cookie), add a "LastChanged" claim containing the current value. When the database changes, the "LastChanged" value should be updated.
+One approach to cookie validation is based on keeping track of when the user database has been changed. If the database hasn't been changed since the user's cookie was issued, there is no need to re-authenticate the user if their cookie is still valid. To implement this scenario, the database stores a `LastUpdated` value. When any user is updated in the database, the `LastUpdated` value is set to the current time.
```

Comment:
@HaoK

Reply:
This is reasonable, identity's security stamp works a bit differently, but this should work too. Basic idea is you just need to check a claim against something in the database that changes when you want to invalidate the user's cookie.  LastUpdated seems fine as a choice, (identity generates a new guids for the stamps)